src/content.ts

```diff
@@ -1638,17 +1638,22 @@ function openInfoMenu() {
             popup.innerHTML = htmlData;
 
             //close button
-            const closeButton = document.createElement("div");
-            closeButton.innerText = chrome.i18n.getMessage("closePopup");
-            closeButton.classList.add("smallLink");
+            const closeButton = document.createElement("button");
+            const closeButtonIcon = document.createElement("img");
+            closeButtonIcon.src = chrome.extension.getURL("icons/close.png");
+            closeButtonIcon.width = 16;
+            closeButtonIcon.height = 16;
+            closeButton.appendChild(closeButtonIcon);
+            closeButton.setAttribute("title", chrome.i18n.getMessage("closePopup"));
+            closeButton.classList.add("sbCloseButton");
             closeButton.setAttribute("align", "center");
             closeButton.addEventListener("click", closeInfoMenu);
             // Theme based color
```

Comment:
Probability should not change colour based on theme anymore

Reply:
Is this a change for this PR?

Reply:
Yes, otherwise the close button will be white on light theme, even though it is inside the SB popup which is always dark

Reply:
Thanks, I've removed this.